torch/fx/experimental/proxy_tensor.py

```diff
@@ -163,19 +184,77 @@ def extract_val(val):
     elif isinstance(val, (int, float, bool)):
         return val
 
+class PythonKeyTracer(Tracer):
```

Comment:
Could you not move this tracer code? It is little hard to see what is changed. Since this is very critical code, we should be careful